### PR TITLE
Requesting Notification Permission

### PR DESCRIPTION
**Pull request by rammadogs**
My fork adds the ability to request permissions to manage notifications, which allows the app to do things such as switching off do not disturb mode.

### CODE REVIEW

File: permission/build.gradle [hunk 1]
```diff
@@ -1,5 +1,8 @@
 apply plugin: 'com.android.library'
 apply from: 'maven-push.gradle'
+apply plugin: 'com.github.dcendents.android-maven'
+
+group='com.github.rammadogs'
```
Comment (k0shk0sh) [inline permission/build.gradle:5]: May I know wht did you add this?
Could you please remove everything related

File: build.gradle [hunk 2]
```diff
@@ -5,6 +5,7 @@ buildscript {
         jcenter()
     }
     dependencies {
+        classpath 'com.github.dcendents:android-maven-gradle-plugin:1.4.1'
```
Comment (k0shk0sh) [inline build.gradle:8]: This should be removed as well

File: permission/build.gradle [hunk 3]
```diff
@@ -1,5 +1,8 @@
 apply plugin: 'com.android.library'
 apply from: 'maven-push.gradle'
+apply plugin: 'com.github.dcendents.android-maven'
```
Comment (k0shk0sh) [inline permission/build.gradle:3]: This as well

**Review by k0shk0sh — CHANGES_REQUESTED** [review]
Please fix the unwanted stuff

**Comment by rammadogs** [comment]
Those changes were necessary to make it work with Jitpack. I need that working for a university assignment so when the assignment's over in a couple of weeks, I'll make the changes :)

**Comment by k0shk0sh** [comment]
If that's the case then use ur own fork for your assignment and then resubmit another pr. I can't accept this.

**Pull request closed without merging**